src/main/kotlin/pmeet/pmeetserver/common/ErrorCode.kt

```diff
@@ -31,6 +31,7 @@ enum class ErrorCode(private val code: String, private val message: String) {
   RESUME_UPDATE_FORBIDDEN("RESUME-40301", "해당하는 이력서를 수정할 권한이 없습니다."),
   RESUME_DELETE_FORBIDDEN("RESUME-40302", "해당하는 이력서를 삭제할 권한이 없습니다."),
   RESUME_COPY_UNAUTHORIZED("RESUME-40003", "해당하는 이력서를 복사할 권한이 없습니다."),
+  RESUME_ACTIVE_CHANGE_UNAUTHORIZED("RESUME-40004", "해당하는 이력서의 프미팅 상태를 변경할 권한이 없습니다."),
```

Comment:
```suggestion
  RESUME_ACTIVE_CHANGE_UNAUTHORIZED("RESUME-40304", "해당하는 이력서의 프미팅 상태를 변경할 권한이 없습니다."),
```
Forbidden에 해당하는 403으로 맞추는 게 좋을 것 같네요.